packages/web-api-scan-runner/src/runner/runner.ts

```diff
@@ -37,25 +37,31 @@ export class Runner {
 
     public async run(): Promise<void> {
         let browser: Browser;
-        let pageScanResult: OnDemandPageScanResult;
         const scanMetadata = this.scanMetadataConfig.getConfig();
 
         const scanStartedTimestamp: number = Date.now();
         const scanSubmittedTimestamp: number = this.guidGenerator.getGuidTimestamp(scanMetadata.id).getTime();
 
         this.logger.logInfo(`Reading page scan run result.`);
-        pageScanResult = await this.onDemandPageScanRunResultProvider.readScanRun(scanMetadata.id);
-        this.logger.setCustomProperties({ scanId: scanMetadata.id, batchRequestId: pageScanResult.batchRequestId });
+        this.logger.setCustomProperties({ scanId: scanMetadata.id, batchRequestId: scanMetadata.batchRequestId });
 
         this.logger.trackEvent('ScanTaskStarted', undefined, { scanWaitTime: (scanStartedTimestamp - scanSubmittedTimestamp) / 1000 });
 
         this.logger.logInfo(`Updating page scan run result state to running.`);
-        pageScanResult = this.resetPageScanResultState(pageScanResult);
-        pageScanResult = await this.onDemandPageScanRunResultProvider.updateScanRun(pageScanResult);
+        const pageScanResult: PartialOnDemandPageScanResult = {
```

Comment:
probably we do not need that strong `PartialOnDemandPageScanResult` type, having `Partial<OnDemandPageScanResult>` on `onDemandPageScanRunResultProvider.updateScanRun()` definition will suffice